.github/workflows/update-beats.yml

```diff
@@ -28,22 +28,25 @@ jobs:
       - uses: actions/checkout@v4
         with:
           ref: ${{ matrix.branch }}
+
       - uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
-      - uses: elastic/apm-pipeline-library/.github/actions/updatecli@current
+
+      - uses: elastic/oblt-actions/updatecli/run-and-notify@v1
```

Comment:
Using `v1` for now until we can support a semver strategy properly.

Reply:
You can use [v0.1.0](https://github.com/elastic/oblt-actions/releases/tag/v0.1.0) if you want to use SemVer

Reply:
But I'm also planning to do a `v1.0.0` release

Reply:
> You can use [v0.1.0](https://github.com/elastic/oblt-actions/releases/tag/v0.1.0) if you want to use SemVer

I won't because the `run-and-notify` uses `slack/send@v1` and `updatecli/run@v1`.

We might need to figure out how to solve this

Reply:
Ah, true